src/grammar.coffee

```diff
@@ -98,8 +98,9 @@ class Grammar
   # * `ruleStack` An {Array} of rules representing the tokenized state at the
   #   end of the line. These should be passed back into this method when
   #   tokenizing the next line in the file.
-  tokenizeLine: (line, ruleStack, firstLine=false, compatibilityMode=true) ->
+  tokenizeLine: (fullLine, ruleStack, firstLine=false, compatibilityMode=true) ->
```

Comment:
Forgot to rename the variable in the docs above: https://github.com/atom/first-mate/pull/67/files#diff-30b5df382fb36749d0d1d282b2df9a53R82